modules/installation-mirror-repository.adoc

```diff
@@ -48,13 +87,51 @@ the pull secret for your mirror registry that you created.
 <6> The release mirror. For a production release, you must specify
```

Comment:
For the following lines:
> <5> For <path_to_pull_secret>, specify the absolute path to and file name of the pull secret for your mirror registry that you created.

The file not only need the pull secret for your mirror registry that you created, but also the pull secret from the OpenShift Infrastructure Providers page. So the description here is not accurate. The description in 'Prerequisites' of disconnected install doc maybe more clear:

> You downloaded the pull secret from the OpenShift Infrastructure Providers page and modified it to include authentication to your mirror repository.

Reply:
https://docs.openshift.com/container-platform/4.2/installing/installing_restricted_networks/installing-restricted-networks-preparations.html#installation-local-registry-pull-secret_installing-restricted-networks-preparations
So, should this section be replaced with this section instead? https://github.com/openshift/openshift-docs/pull/16678/files#diff-caf7c18be6cf05b5acd8148e443eb3e2R1

Reply:
For disconnected clusters, we'll need different authorities in the pull secret for different steps (some previous discussion [here][1]).  For mirroring from Quay to the mirror registry (which may be what this step is?), you'll need `quay.io` and your mirror token.  For launching the cluster from your mirror registry, you'll only need the mirror token.  For pushing telemetry from the cluster (only when you have network access to us, possibly via a proxy), you'll need ... `cloud.openshift.com`?  Etc.  But it may be easier to just have a secret with the mirror token and the cloud.openshift.com-sourced tokens, and use that everywhere without trying to break it down to minimal sets for each action.

[1]: https://github.com/openshift/openshift-docs/pull/16678#discussion_r325832486

Reply:
https://docs.openshift.com/container-platform/4.2/installing/installing_restricted_networks/installing-restricted-networks-preparations.html#installation-local-registry-pull-secret_installing-restricted-networks-preparations looks good to me, Should NOT be replaced.  Just like what wking explained, At this step, user is going to mirror release image, so need auth for both Quay and mirror registry token.

Reply:
@jianlinliu, doesn't that just give you the mirror registry token? I'm proposing changing to https://osdocs628--ocpdocs.netlify.com/openshift-enterprise/latest/installing/installing_restricted_networks/installing-restricted-networks-preparations.html#installation-adding-registry-pull-secret_installing-restricted-networks-preparations, which has the same step to generate the mirror registry token and then add it to the pull secret that you get from the OpenShift Infrastructure Providers page.

Reply:
Oops, I missed your new change. Thanks for pointing the url to me. Most looks good to me except `registry.redhat.io` pull secret name looks like a bit weird.